pages/login_page.py

```diff
@@ -0,0 +1,34 @@
+from pages.base_page import BasePage
+from selenium.webdriver.common.by import By
+
+sign_field_selector = (By.ID, 'sign')
+password_field_selector = (By.ID, 'password')
+button_sign_selector = (By.ID, 'signbutton')
+
+LOGIN = '89994516217'
+PASSWORD = 'Artyom#Access!'
+
+class Login_page(BasePage):
+    def __init__(self, browser):
+        super().__init__(browser)
+
+    def find_sign_field(self):
+        return self.find(sign_field_selector)
```

Comment:
Autotests are really fast. What if you try to find something, but page are not fully loaded? Or element appears on page after some time or by user interaction?